_ml-commons-plugin/cluster-settings.md

```diff
@@ -47,49 +45,44 @@ plugins.ml_commons.task_dispatch_policy: round_robin
 
 ## Set sync up job intervals 
 
+When returning runtime information with the [profile API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#profile), ML Commons will run a regular sync up job to sync up newly loaded or unloaded models on each node. When set to `0`, ML Commons immediately stops sync up jobs.
+
 ### Setting
 
 ```
 plugins.ml_commons.sync_up_job_interval_in_seconds: 10
 ```
 
-### Description
-
-When returning runtime information with the [profile API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#profile), ML Commons will run a regular sync up job to sync up newly loaded or unloaded models on each node. When set to `0`, ML Commons immediately stops sync up jobs.
-
 ### Values
 
 - Default value: `10`
 - Value range: [0, 86,400]
 
 ## Predict monitoring requests
```

Comment:
```suggestion
## Monitoring predict requests
```

Reply:
Is there a reason why monitoring has to appear first? This is meant to be in line with our style and standard for headings?